### DISCUSSION TITLE
Shared model directories

### DISCUSSION THREAD

**Discussion by krummrey** [lllyasviel/stable-diffusion-webui-forge#198] [Q&A]
Hi,
is there a way to set a base directory that all other model directories are based off? Or do I need to specify each and every directory for all models individually? It adds up with all the upscalers and such. Also all controlnet models can't be loaded from a different directory? Or do I have to set hardlinks which has been problematic with github in the past.

**Comment by huchenlei** [comment]
After #203, you can use `--forge-ref-a1111-home` commandline arg to reference models in existing A1111 checkout.
